Integrations/VulnDB/VulnDB.py

```diff
@@ -0,0 +1,431 @@
+        'Keywords': vuln.get('keywords', ''),
+        'PublishedDate': vuln.get('vulndb_published_date', '').rstrip('Z'),
+        'TDescription': vuln.get('t_description', ''),
+        'SolutionDate': vuln.get('solution_date', '').rstrip('Z'),
+        'DiscoveryDate': vuln.get('disclosure_date', '').rstrip('Z'),
+        'ExploitPublishDate': vuln.get('exploit_publish_date', '').rstrip('Z'),
+    }
+
+    cve_ext_reference_values = [ext_reference['value'] for ext_reference in
+                                vuln.get('ext_references', [])]
+
+    cvss_metrics_details = [{
+        'Id': cvss_metrics_data.get('id', 0),
+        'AccessVector': cvss_metrics_data.get('access_vector', ''),
+        'AccessComplexity': cvss_metrics_data.get('access_complexity', ''),
+        'Authentication': cvss_metrics_data.get('authentication', ''),
+        'ConfidentialityImpact': cvss_metrics_data.get('confidentiality_impact', ''),
+        'IntegrityImpact': cvss_metrics_data.get('integrity_impact', ''),
+        'AvailabilityImpact': cvss_metrics_data.get('availability_impact', ''),
+        'GeneratedOn': cvss_metrics_data.get('generated_on', ''),
+        'Score': cvss_metrics_data.get('score', 0),
+    } for cvss_metrics_data in vuln['cvss_metrics']]
+
+    vendor_details = [{'Id': vendor.get('vendor', {'id': 0})['id'], 'Name': vendor.get('vendor', {'name': ''})['name']}
+                      for vendor in vuln['vendors']]
+
+    product_details = []
+    for product in vuln['products']:
+        product_versions = [{'Id': version.get('id', ''), 'Name': version.get('name', '')} for version in
+                            product.get('versions', [])]
+        product_details.append({
+            'Id': product.get('id', ''),
+            'Name': product.get('name', ''),
+            'Versions': product_versions
+        })
+
+    default_classification = {'longname': '', 'description': ''}
+    classification_details = [{'Longname': classification.get('classification', default_classification)['longname'],
+                               'Description': classification.get('classification', default_classification)[
+                                   'description']}
+                              for classification in vuln['classifications']]
+
+    return {
+        'Vulnerability': vulnerability_details,
+        'CVE-ExtReference': {
+            'Value': cve_ext_reference_values
+        },
+        'CvssMetrics': cvss_metrics_details,
+        'Vendor': vendor_details,
+        'Products': product_details,
+        'Classification': classification_details
+    }
+
+
+def vulndb_vulnerability_results_to_demisto_results(res):
+    if 'error' in res:
+        return_error(res['error'])
+    else:
+        if 'vulnerability' in res:
+            results = [res['vulnerability']]
+        elif 'results' in res:
+            results = res['results']
+        else:
+            demisto.results({
```

Comment:
`demisto.results()` doesn't `return`, it seems like you need to add a return after it